### PR TITLE
Allow passing in Secret with TLS data for Elasticsearch

### PR DESCRIPTION
**Pull request by leifmadsen**

### CODE REVIEW

**Comment by leifmadsen** [comment]
[Investigate alternative ElasticSearch Operators for better supportability over abandoned ECL ES Operator](https://trello.com/c/3MLo7kUk/854-investigate-alternative-elasticsearch-operators-for-better-supportability-over-abandoned-ecl-es-operator)

File: roles/smartgateway/defaults/main.yml [hunk 1]
```diff
@@ -3,3 +3,5 @@
 size: 1
 container_image_path: quay.io/redhat-service-assurance/smart-gateway:latest
 prometheus_scrape_interval: 1s
+use_basic_auth: false
```
Comment (csibbitt) [inline roles/smartgateway/defaults/main.yml]: Should we default this to true now; or at least soon after all the ECK code merges?

Comment (leifmadsen) [reply roles/smartgateway/defaults/main.yml]: Well, the defaults here are basically for non-SAO managed stand ups (at least that's what I've been thinking in my head). If someone were to stand up a SG (for some strange reason) independent of the SAO, I would somewhat expect it to "just work" (which is not a tested scenario currently).

The SAO will pass all values it requires to the SGO via the `SmartGateway` object, so will override this to `use_basic_auth: true`. You can see that here:  https://github.com/redhat-service-assurance/service-assurance-operator/pull/27/files#diff-20856cacc013bbb9e77b19cced6ac91aR28

Basically, think of the defaults here as non-SAF defaults.
